ontology.ttl

```diff
@@ -0,0 +1,212 @@
+@prefix : <http://nva.unit.no/ontology#> .
+@prefix datacite: <http://purl.org/spar/datacite/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Publication a rdfs:Class ;
+  rdfs:comment "Base concept for all items in NVA" .
+
+# Publication-related concepts
+
+:EntityDescription a rdfs:Class ;
+  rdfs:comment "A description of Publication" .
+
+:Contributor a rdfs:Class ;
+  rdfs:comment "A Contributor of a Publication" .
+
+:Organization a rdfs:Class ;
+  rdfs:comment "An Organization" .
+
+:Publisher a rdfs:Class ;
+  rdfs:comment "A publishing organization" ;
+  rdfs:subClassOf :Organization .
+
+:Role a rdfs:Class ;
+  rdfs:comment "A Role in the publication process" .
+
+:PublicationType a rdfs:Class ;
+  rdfs:comment "Base class for Publication types" .
+
+:JournalArticle a rdfs:Class ;
+  rdfs:subClassOf :PublicationType ;
+  rdfs:comment "A publication type that is published in an academic Journal" .
+
+:License a rdfs:Class ;
+  rdfs:comment "A license describing the legal rights associated with a Publication" .
+
+:Date a rdfs:Class ;
+  rdfs:comment "A date associated with a Publication" .
+
+# Publication status concepts
+
+:Status a rdfs:Class ;
+  rdfs:comment "A status of a Publication in the NVA workflow" .
+
+:PublishedStatus a rdfs:Class ;
+  rdfs:subClassOf :Status ;
+  rdfs:comment "The status of a published Publication" .
+
+# Files
+
+:FileSet a rdfs:Class ;
+  rdfs:comment "The set of Files associated with a Publication" .
+
+:File a rdfs:Class ;
+  rdfs:comment "A File associated with a Publication" .
+
+# Roles
+
+:CreatorRole a rdfs:Class ;
+  rdfs:comment "The base class for all creator roles" ;
+  rdfs:subClassOf :Role .
+
+:ContributorRole a rdfs:Class ;
+  rdfs:comment "The base class for all contributor roles" ;
+  rdfs:subClassOf :Role .
+
+# Relations
+# Basic relations
+
+:hasIdentifier a rdf:Property ;
```

Comment:
What is the scope of an identifier? (We have to keep track of the identifying system the identifier refers to)